profiles/desktop.py

```diff
@@ -1,6 +1,5 @@
 # A desktop environment selector.
 import archinstall
-from archinstall.lib.menu import Menu
```

Comment:
Don't forget to change https://github.com/archlinux/archinstall/blob/bdcf33cdc6f6ac4c0bcc8610c064b47a82284eb9/profiles/desktop.py#L46

Into `archinstall.Menu` : )

Reply:
I like the recent changes!

Reply:
Yes missed that